src/compile/layout/header.ts

```diff
@@ -84,6 +84,28 @@ export function getTitleGroup(model: Model, channel: HeaderChannel) {
   };
 }
 
+export function getLabelAlign(angle: number) {
```

Comment:
Let's call it labelAlign as we don't really add get prefix in the rest of the code base. Same for labelBaseline